### ISSUE TITLE
Emulation freezes when sound device is removed

### ISSUE THREAD

**Issue by toxeia** [project64/project64#2159]
I've narrowed the cause down to Windows sound devices being removed during emulation. I use Creative Pebble V3 speakers most of the time. When the volume knob is lowered enough the speakers turn off, cutting the USB connection and effectively disconnecting the speakers. 

This results in emulation hangs immediately. I haven't found a way to resume emulation when this occurs.

Possibly worth noting: if I start emulation with the speakers off and then turn them on, emulation stutters but successfully switches output when Windows handles that automatically. Still breaks if the speakers get turned off though.

**Comment by Squall-Leonhart** [comment]
use the latest azimer audio with xaudio.

**Comment by toxeia** [comment]
Is the latest v0.70 WIP 10? Just tried it, it also freezes. In line with trying other plugins, I tried the Jabo's DirectSound 1.7.0.7. This plugin experiences stutters when the sound device is removed but won't freeze. This solves the largest issue, but even after putting the sound device back it will take a minute for Project64 to have sound again. Seems like clicking on and off the window a few times will help. 

I'll stick with Jabo's for now - losing sound for a minute or so is fine compared to losing hours of progress because I had failed to save.
